app/assets/stylesheets/modules/_forms.scss

```diff
@@ -54,3 +54,7 @@ span[title="required"] {
 .cfp-date-input {
   width: 50%;
 }
+
+input.boolean {
+  margin: 4px;
+}
```

Comment:
Really glad you did this, that was annoying me to no end

Reply:
😄